ansible/roles/stack-sunbird/templates/dhiti-service.env

```diff
@@ -0,0 +1,85 @@
+APPLICATION_PORT_NUMBER={{dhiti_application_port_number}}
+APPLICATION_HOST_NAME={{dhiti_application_host_name}}
+APPLICATION_BASE_URL={{dhiti_application_base_url}}
+NODE_ENV={{dhiti_node_env}}
+APIDOC_URL={{dhiti_apidoc_url}}
+APIDOC_PATH={{dhiti_apidoc_path}}
+
+#Cloud storage
+CLOUD_STORAGE={{dhiti_cloud_storage}}
+BUCKET_NAME={{dhiti_bucket_name}}
+
+#Druid configuration
+DRUID_HOST={{sunbird_druid_broker_host}}
+DRUID_PORT={{dhiti_druid_port | default("8082")}}
+OBSERVATION_DATASOURCE_NAME={{dhiti_observation_datasource_name | default("dev_sl_observations")}}
+ASSESSMENT_DATASOURCE_NAME={{dhiti_assessment_datasource_name | default("dev_sl_assessments")}}
+ENROLLMENT_DATASOURCE_NAME={{dhiti_enrollment_datasource_name | default("dev_sl_enrollment")}}
+TELEMETRY_DATASOURCE_NAME={{dhiti_telemetry_datasource_name | default("dev_sl_telemetry")}}
+EVIDENCE_DATASOURCE_NAME={{dhiti_evidence_datasource_name | default("dev_sl_observation_evidences")}}
+SURVEY_DATASOURCE_NAME={{dhiti_survey_datasource_name | default("dev_sl_survey_druid")}}
+SURVEY_EVIDENCE_DATASOURCE_NAME={{dhiti_survey_evidence_datasource_name | default("dev_sl_survey_evidences")}}
+OBSERVATION_EVIDENCE_DATASOURCE_NAME={{dhiti_observation_evidence_datasource_name | default("dev_sl_observation_evidences")}}
+CONTENT_REPORT_THRESHOLD={{dhiti_content_report_threshold | default("5")}}
+ENTITY_SCORE_REPORT_THRESHOLD={{dhiti_entity_score_report_threshold | default("5")}}
+OBSERVATION_SCORE_REPORT_THRESHOLD={{dhiti_observation_score_report_threshold | default("2")}}
+ASSESSMENT_SUBMISSION_REPORT_THRESHOLD={{dhiti_assessment_submission_report_threshold | default("3")}}
+EVIDENCE_THRESHOLD={{dhiti_evidence_threshold | default("3")}}
+
+#Auth Config
+sunbird_keycloak_auth_server_url={{keycloak_auth_server_url}}
+sunbird_keycloak_realm={{keycloak_realm | default("sunbird")}}
+sunbird_keycloak_client_id={{sunbird_keycloak_client_id | default("portal")}}
+sunbird_keycloak_public={{sunbird_keycloak_public | default("true")}}
+sunbird_cache_store={{sunbird_cache_store | default("")}}
+sunbird_cache_ttl={{dhiti_sunbird_cache_ttl | default("1800")}}
+
+URL_PREFIX={{dhiti_url_prefix | default("api/v1")}}
+
+#ASSESSMENT service
+ASSESSMENT_SERVICE_APPLICATION_ENDPOINT=http://sl-assessments-service:4201.{{namespace}}.svc.cluster.local
+ASSESSMENT_SERVICE_BASE_URL={{dhiti_assessment_service_base_url | default("/assessment/")}}
+
+#KENDRA service
+KENDRA_APPLICATION_ENDPOINT=http://kendra-service:4202.{{namespace}}.svc.cluster.local
+KENDRA_BASE_URL={{dhiti_kendra_base_url | default("/kendra/")}}
+
+#Gotenberg configuration
+GOTENBERG_URL={{dhiti_gotenberg_url | default ("http://gotenberg-service:3000/convert/html")}}
+
+#Highchart configuration
+HIGHCHART_URL=http://highcharts-service:8080.{{namespace}}.svc.cluster.local
+
+# FILE PATHS FOR PORTAL REPORTS
```

Comment:
What can be the sample values for these set of variables? Please note, we want to define as minimum number of variables in private as posible. Can these have defaults?

Reply:
Every Managed Learn Service is using lot of env vaiables